vllm/sampling_params.py

```diff
@@ -94,6 +99,9 @@ def _verify_args(self) -> None:
         if not -2.0 <= self.frequency_penalty <= 2.0:
             raise ValueError("frequency_penalty must be in [-2, 2], got "
                              f"{self.frequency_penalty}.")
+        if self.repetition_penalty <= 0.0:
+            raise ValueError("repetition_penalty must be a strictly positive "
+                             f"float, got {self.repetition_penalty}.")
```

Comment:
```suggestion
        if self.repetition_penalty <= 0.0:
            raise ValueError("repetition_penalty must be a strictly positive "
                             f"float, got {self.repetition_penalty}.")
        if self.repetition_penalty != 1.0 and (
                abs(self.frequency_penalty) > _SAMPLING_EPS
                or abs(self.presence_penalty) > _SAMPLING_EPS):
            raise ValueError(
                f"repetition_penalty cannot be used with "
                f"frequency_penalty and presence_penalty."
                f"got repetition_penalty={self.repetition_penalty}, "
                f"frequency_penalty={self.frequency_penalty}, "
                f"presence_penalty={self.presence_penalty}")
```


It seems that repetition penalty and other penalties cannot be used together, how about exception handling like above?